File: JS/Basics/First Steps/Exercise/basketballEquipment.js
function basketballEquipment(input) {
    let annualTax = Number(input[0]);
    let basketsPrice = annualTax * 0.6;
    let outfitPrice = basketsPrice * 0.8;
    let ballPrice = 1 / 4 * outfitPrice;
    let accessoriesPrice = 1 / 5 * ballPrice;

    let totalPrice = annualTax + basketsPrice + outfitPrice + ballPrice + accessoriesPrice;

    console.log(totalPrice);
}

basketballEquipment(['365']);

File: Python/SoftUni PB/Exercises/PB Online Exams/PB Exams 9 and 10 March 2019 COMPLETED/basketball_equipment.py
annual_tax = int(input())

basketball_sneakers = 0.6 * annual_tax
basketball_outfit = 0.8 * basketball_sneakers
basketball_ball = 1/4 * basketball_outfit
basketball_accessories = 1/5 * basketball_ball

total_price = basketball_sneakers + basketball_outfit + basketball_ball + basketball_accessories + annual_tax

print(f"{total_price:.2f}")
